include/net/net_context.h

```diff
@@ -66,8 +66,10 @@ struct net_context;
  * @typedef net_context_recv_cb_t
  * @brief Network data receive callback.
  *
- * @details The recv callback is called after a network data is
- * received.
+ * @details The recv callback is called after a network data is received.
```

Comment:
this should read either "after network data is received" or maybe "after a network data packet is received"